lang/LangPrimSource/PyrFilePrim.cpp

```diff
@@ -1326,6 +1326,67 @@ int prPipeOpen(struct VMGlobals *g, int numArgsPushed)
 	return errNone;
 }
 
+int prPipeOpenArgv(struct VMGlobals *g, int numArgsPushed)
+{
+	PyrSlot *callerSlot, *argsSlot, *modeSlot;
+	char mode[12];
+
+	callerSlot = g->sp - 2;
+	argsSlot = g->sp - 1;
+	modeSlot = g->sp;
+
+#ifdef SC_IPHONE
```

Comment:
this needs an explanatory comment

Reply:
Doing a git blame apparantly points to me as the author of these lines, but I have no idea why they are there, most probably someone told me that I should do this check. I imagine that on ios you can't fork ? Anyone else knows more about this ?